ui/src/components/admin/faqs/FaqGrid.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { AgGridReact } from 'ag-grid-react';
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
+import { Heading, Container, Switch, HStack } from '@chakra-ui/react';
+
+export default function FaqGrid(props) {
+  const { faqs, setSelected } = props;
```

Comment:
can combine this to be `export default function FaqGrid({ faqs, setSelected }) {...`

Reply:
i think the convention is to use props hehe, so maybe we can keep it as props instead?

Reply:
can too idm. i just like shorter code hahha